### ISSUE TITLE
Accessibility: Allow people to show a volume control Quick Action button in the Action Center so it's easier to change the volume.

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/Retiled#125]
Ideally this would expand into a "device control" Quick Action button like on iOS (along with an option to always show it on screen and allow dragging it around), but for now I think volume is a very frequently-used option, so it should exist first. Once the device control support is added, there should be a way to let people set a default command when tapping its Quick Action button or open the menu, and have the long-press always open the menu. That way people can still have the volume control UI open on a tap while adding a long-press for the rest unless they change it.
